Saudox/app/Http/Controllers/Profissional/ProfissionalController.php

```diff
@@ -88,13 +88,49 @@ public function salvarCadastrarPaciente(Request $request) {
             'lt' => 'O campo :attribute deve ser menor que 100.',
             'unique' => 'O :attribute já está cadastrado',
         ];
-        $time = strtotime($entrada['data_nascimento']);
-        $entrada['data_nascimento'] = date('Y-m-d',$time);
 
         if($entrada['lista_irmaos'] == "") {
             $entrada['lista_irmaos'] = "Nenhum";
         }
 
+        $rota_erro = "profissional.criar_paciente";
+
+        if(!isset($entrada["data_nascimento"])
+            || $entrada["data_nascimento"] == ""
+            || $entrada["data_nascimento"] == NULL
+            || !strtotime($entrada["data_nascimento"])
+            || !$this->validatarData($entrada["data_nascimento"])
+        ) {
+
```

Comment:
O 
```php
!$this->validatarData($entrada["data_nascimento"])
```
já faz vários dos trabalhos a cima, não?

Reply:
não, se for vazio, por exemplo, ele dá 1 primeiro de janeiro de 1970, mas vou mover isso tudo pra dentro da função ainda assim